development/build/static.js

```diff
@@ -15,11 +15,21 @@ module.exports = function createStaticAssetTasks({
   livereload,
   browserPlatforms,
   shouldIncludeLockdown = true,
+  shouldIncludeSnow = true,
```

Comment:
all changes made to this entire file are explained [here](https://github.com/MetaMask/metamask-extension/pull/15580#:~:text=NOTICE%20%2D%20To%20this%20PR%20was%20merged%20%235%20which%20separately%20introduces%20support%20for%20changing%20build%20process%20behaviour%20based%20on%20the%20browser%20being%20built%20%2D%20read%20more%20about%20it%20in%20the%20PR%20description)